### PR TITLE
Add the draft for project design overview

### LINKED ISSUE #1395: Introduction to the "Project Design Handbook"
### Summary

Following a short meeting between Malvika Sharan, Yasemin Turkyilmaz-van Velden @yaseminturkyilmaz and myself (Nicolas Dintzner @NZR), we thought we could get started with the introduction to the "Project Design Handbook".  

The objective is to provide a project overview, in chronological order (from preparation to archival), with key steps and references to the relevant section of the book. 

### What needs to be done?

- [ ]  Create a "bullet-point" list of the key steps identified in the following publication (https://www.sciencedirect.com/science/article/pii/S2666389920301331)  
- [ ]  Provide minimal, yet relevant, highlight of each point (based on the content of the publication)
- [ ]  link each item to the right chapter
- [ ]  any further clean-up (as the content found in the paper may not be always ideal for the book! Up for discussion)

### Who can help?

* @marianaiv is attempting the first draft - Book Dash attendees are welcome to collaborate with her
* *Nicolas and Yasemin will be tagged for review*
* *We will need assistance for the consistency checks + formatting/content review by Turing way people :)*

---

### Updates

None yet!

### PR DESCRIPTION
**Pull request by malvikasharan**
### Summary

Fixes #1395

* Add the draft by Mariana from her HackMD along with my notes for the initial chapters

### List of changes proposed in this PR (pull-request)

* Adding introduction
* Adding notes from CW21 workshop
* Adding references
* Adding notes from HackMD: https://hackmd.io/d7zEZo25Soq1mOyJgz5vCQ?both

### What should a reviewer concentrate their feedback on?

- [x] Covers all topics
- [x] Refers to different chapters
- [x] Everything looks ok?

### Acknowledging contributors

- [x] All contributors to this pull request are already named in the [table of contributors](https://github.com/alan-turing-institute/the-turing-way/blob/master/README.md#contributors) in the README file.
- [x] Add @marianaiv

### CODE REVIEW

**Review by ninadicara — COMMENTED** [review]
Hiya, I've got as far as the repo chapter for now - so much work and writing has gone into this!  
Most of the comments I've left are really minor things so I've put them as suggested changes. Hope that's okay!
I've really enjoyed reading it so far - it's great having so many links out to other sections ✨

File: book/website/project-design/pd-overview.md [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+
+This lack of planning contributes to the fact that most research work can not be directly and independently reproduced, and that communication and collaboration style across different groups differ and hence are challenging.
+
+To help learn good practices, *The Turing Way* provides various chapters for {ref}`reproducibility<rr>`, {ref}`communication<cm>` and {ref}`collaboration<cl>` that we consider essential for research reproducibility.
+Although publications{cite}`Turkyilmaz-vanderVelden2020projectdesign` and _The Turing Way_ chapters on specific methods, tools and practices exist -- it can be overwhelming to know which chapters to read if you don't already know about the concepts.
+
+In this chapter, we have curated essential practices and recommendations and linked them to individual chapters across different guides.
+
+```{note}
+There are many chapters that we don't link here to avoid cognition load for readers who are new to reproducible project design.
```
[the remark this answers is not in the document]
Comment (malvikasharan) [reply book/website/project-design/pd-overview.md:35]: That’s such a good catch - a phrase with lots of cognition load 🙈 I like your alternative suggestion.

File: book/website/project-design/pd-overview/pd-overview-repro.md [hunk 2]
```diff
@@ -0,0 +1,95 @@
...
+
+```{note}
+**I work alone, do I need to think about project design?**
+
+The short answer is 'yes'.
+Project design will allow you to manage your work well for yourself (see the section: {ref}`Getting Started<pd-overview-planning-started>`).
+
+A little work and time investment early on in project design saves a lot of time later when any circumstances that demand change.
+It is really hard for a project to move from practices that were designed for one person to practices that work for a team.
+Therefore, it is essential to document and use practices that will enable collaboration if an when you have to involve others in your project.
```
Comment (malvikasharan) [inline book/website/project-design/pd-overview/pd-overview-repro.md:84]:
```suggestion
Therefore, it is essential to document and use practices that will enable collaboration if and when you have to involve others in your project.
```

File: book/website/project-design/pd-overview.md [hunk 3]
```diff
@@ -0,0 +1,40 @@
+(pd-overview)=
+# Overview of Project Design
+
+## Summary
+
+Project design techniques can help researchers clearly identify and communicate their project goals, skill requirements and resource available to them.
+It ensures that all stakeholders can work together efficiently, apply reproducibility methods and communicate their work effectively with their target audience.
+A project design requires project leads, managers and organisers to be deliberate and clear about their expectations from the beginning of their projects to ensure successful implementation of their plans at all stages of research.
+
+```{figure} ../figures/ResearchCycle.jpg
```
Comment (malvikasharan) [inline book/website/project-design/pd-overview.md:10]:
```suggestion
```{figure} ../figures/research-cycle.jpg
```

File: book/website/project-design/pd-overview/pd-overview-sharing.md [hunk 4]
```diff
@@ -0,0 +1,68 @@
...
+
+A tool that can be used to avoid this is [protocols.io](https://www.protocols.io/).
+It provides a way to ensure that your protocols are openly available, allowing you to update them while keeping track of the changes.
+Furthermore, having your protocols online makes them easier to share, creating opportunity for contributing.
+
+## Share Analysis Scripts and Research Software
+
+If you have been using a version control system you will have this covered.
+
+If developing analysis scripts and research software is part of your research, you need to use a version control system.
```
Comment (ninadicara) [inline book/website/project-design/pd-overview/pd-overview-sharing.md:37]: Maybe link to the Version Control chapter here?

```suggestion
If developing analysis scripts and research software is part of your research, you need to use a version control system. 
If you are new to version control you can read more about it in the {ref}`Version Control<rr-vcs>` chapter.
```

File: book/website/project-design/pd-overview/pd-overview-sharing.md [hunk 5]
```diff
@@ -0,0 +1,68 @@
...
+This automatically makes it easier to share and makes it citable.
+You can read about DOIs in the chapter on {ref}`Making Research Components Citable<cm-citable>`.
+
+## Collecting your Research
+
+Once you are done with your research you may want to collect all the digital parts of your project in one place.
+This is called a research compendium.
+Publishing your paper along with a research compendium allows the full extent of your research: from the design of the project, through the data recollection and analysis and the resulting outputs.
+
+This has an infinity of advantage, it makes your work shareable and reproducible, others can build upon it and gives you more visibility.
```
Comment (ninadicara) [inline book/website/project-design/pd-overview/pd-overview-sharing.md:50]:
```suggestion
This has endless advantages. It makes your work shareable and reproducible, others can build upon it and gives you more visibility.
```

File: book/website/project-design/pd-overview/pd-overview-sharing.md [hunk 6]
```diff
@@ -0,0 +1,68 @@
...
+Even if you got a license at the beginning of the project you need to think about it again when sharing your outputs and final results - this allows people to have the information about how your research should be reused and shared.
+
+If you want more information about how to choose and add a license to your project you can check the {ref}`Licensing Chapter<rr-licensing>`.
+
+(pd-overview-citation)=
+## Receive Citations
+
+All this hard work will have its reward. Having published all your research from the design to the results adds visibility to your work and more opportunity to get credit.
+
+Not only your results can be cited, but your methods and protocols can be reused and your design can be shared and.
```
Comment (ninadicara) [inline book/website/project-design/pd-overview/pd-overview-sharing.md:66]:
```suggestion
Not only can your results can be cited, but your methods and protocols can be reused and your design can be shared.
```

File: book/website/project-design/pd-overview/pd-overview-version.md [hunk 7]
```diff
@@ -0,0 +1,50 @@
+(pd-overview-version)=
+# Version Control and Documentation
```
Comment (ninadicara) [inline book/website/project-design/pd-overview/pd-overview-version.md:2]: This is a helpful overview of some really crucial concepts!

**Review by ninadicara — APPROVED** [review]
Hi there! Sorry it's taken me so long to finish this review. 
This is such a great section, and really helpful for covering lots of concepts to give someone a cohesive overview and then signposting them to all the relevant chapters so they can read more if they need to. Awesome work! ⭐ 
I've left a couple more comments which are mostly full stops or typos and one or two suggestions to change wording. The only major comment (I think) is about putting the bullet point list in pd-overview-repro.md into some subsections - otherwise I find it can be easy to skim read instead of taking everything in properly, but that's totally up to whether you think that's appropriate there. 
Just let me know if there's anything more I can do :)
Nina

File: book/website/project-design/pd-overview/pd-overview-repro.md [hunk 8]
```diff
@@ -0,0 +1,88 @@
+(pd-overview-repro)=
+# Communication and Collaboration for Reproducibility
+
+Good communication from collaboration practices are complementary to research reproducibility, and often it is hard to separate these concepts from each other.
+In _The Turing Way_, we consider them essential for research reproducibility and provide separate guides for {ref}`communication<cm>` and {ref}`collaboration<cl>`.
+You can visit specific chapters to gain an in-depth understanding and evaluation of which practices meet the specific requirements in your project.
+
+Here, we provide a non-exhaustive list of ways to ensure that you, and everyone else, understand (and communicate about) what the project is about, who the stakeholders are and they can collaborate.
```
[the remark this answers is not in the document]
Comment (malvikasharan) [reply book/website/project-design/pd-overview/pd-overview-repro.md]: This is great suggestion @ninadicara. I have now edited the file to categorise them into 3 sections:

- Documenting project plans and processes for transparency
- Project's "Who-is-Who"
- Participation and contribution process

File: book/website/project-design/pd-overview/pd-overview-sharing.md [hunk 9]
```diff
@@ -0,0 +1,68 @@
...
+# Sharing Your Research Work
+Now that you are in the last phase of your project, you want to make sure that your outputs are available to anyone interested in analysing or reusing them.
+Science can only progress from building on top of each other's results.
+The first step is sharing your work.
+
+But only sharing is not enough.
+You need to make sure that your research is **F**indable, **A**ccesible, **I**noperable and **R**eusable, or **FAIR**.
+This is already taken into account when developing your Data Management Plan, However, you also need to think about it when making your results available.
+
+You can learn more about this in a chapter on {ref}`making data FAIR<rr-rdm-fair>`).
```
[the remark this answers is not in the document]
Comment (malvikasharan) [reply book/website/project-design/pd-overview/pd-overview-sharing.md:13]: Thanks, Nina. I rephrased that section and connected it with the earlier section where it was already introduced.

**Comment by malvikasharan** [comment]
@ninadicara Really helpful suggestions and edits.
I have divided the first chapter into three sections. It does give a better and more cohesive structure to the document.

Thank you so much for such thorough and constructive feedback. 🙏

**Comment by malvikasharan** [comment]
@all-contributors please add @ninadicara for review and edits

**Comment by malvikasharan** [comment]
@all-contributors please add @marianaiv for content

File: book/website/_toc.yml [hunk 10]
```diff
@@ -233,6 +233,19 @@
 
 - file: project-design/project-design
   sections:
+  - title: Overview of Project Design
+    file: project-design/pd-overview
+    sections:
+    - title: Planning for Project Design
+      file: project-design/pd-overview/pd-overview-planning
+    - title: Communication and Collaboration for Reproducibility
```
Comment (malvikasharan) [inline book/website/_toc.yml:241]:
```suggestion
    - title: Communication and Collaboration
```

**Comment by malvikasharan** [comment]
This from my side is ready to merge. @marianaiv please go ahead and merge it when you are ready! 🌷

**Pull request merged**